spec/services/search_spec.rb

```diff
@@ -67,11 +66,9 @@
                                  linkage_group_label: 'linkage group label')
     lg2 = create(:linkage_group, consensus_group_assignment: 'no consensus',
                                  linkage_group_label: 'group2')
-    lm1 = create(:linkage_map, linkage_map_label: 'linkage map label',
-                               map_version_no: '1',
+    lm1 = create(:linkage_map, map_version_no: '1',
                                plant_population: pp1)
-    lm2 = create(:linkage_map, linkage_map_label: 'linkage map label',
-                               map_version_no: '333',
+    lm2 = create(:linkage_map, map_version_no: '333',
                                plant_population: pp1)
     mp1 = create(:map_position, map_position: '102.8',
                                 linkage_group: lg1,
```

Comment:
Why have you changed that file? Were these object violating new uniqueness constraints?